Sources/StreamChat/StateLayer/UserSearch.swift

```diff
@@ -60,6 +60,7 @@ public final class UserSearch {
     // MARK: - Private
     
     private func search(query: UserListQuery, pagination: Pagination) async throws -> [ChatUser] {
```

Comment:
while we're here: `func search(term: String)` the `String` should be optional here.

Reply:
Thank you! Done